merge_test.go

```diff
@@ -1767,7 +1886,7 @@ func TestMergeArray(t *testing.T) {
 				So(err, ShouldBeNil)
 			})
 
-			Convey("throw an error when delete point cannot be found", func() {
+			Convey("don't throw an error when delete point cannot be found", func() {
```

Comment:
In case of `((insert )) `  and `(( delete ))`, it takes those values  always as regex. So you wouldn't find an element with those metacharacters if you don't skip them. 
```yml
jobs:
- name: dea*
```
```yml
(( delete "dea\*" )) 
```